python/ray/tests/test_autoscaler.py

```diff
@@ -340,6 +340,43 @@ def finish_starting_nodes(self):
     "worker_start_ray_commands": [],
 }
 
+TINY_CLUSTER = {
```

Comment:
can we use MULTI_WORKER_CLUSTER instead https://github.com/ray-project/ray/blob/d67c57007ba600a42dd88920292b27a7ade3a8c9/python/ray/tests/test_resource_demand_scheduler.py#L91 ?

Reply:
We could, but MULTI_WORKER_CLUSTER had workers of different types, with different max_workers.
It seems to be more clean and explicit to have a single node type, with max_wokers=min_workers=10

Reply:
given that we're trying to test a behavior that acts on individual node types, would it make sense to have more than one node type in the test?

Reply:
Good point.